### ISSUE TITLE
Changing "CSV dialect options" doesn't affect CSV Mapper

### ISSUE THREAD

**Issue by danielmittag** [zazuko/cube-creator#279]
When CSV dialect options are changed the CSV mapper doesn't refresh. So it actually isn't possible to fix the mapping by specifying the right separator char because these options are only available after having loaded the CSV.

**Comment by martinmaillard** [comment]
I thought this was working. @lucafurrer any idea here? The CSV doesn't seem to be re-parsed when dialect options are changed.

**Comment by lucafurrer** [comment]
I just had a first look at the code, and I suppose it is bad error handling. When I change something not valid, I get an error in the code but it is not raised to the UI.
```typescript
    } catch (err) {
      error(err)
      csvSource.pointer.addOut(schema.error, err.message)
    }
```

**Comment by martinmaillard** [comment]
Ok thanks, I will check the UI. But my guess is that the previously parsed columns don't get deleted when the error happens and that's why the UI is not showing it.

**Comment by lucafurrer** [comment]
Yes I guess so, it would overwrite the columns but if fails while parsing some lines above.

**Comment by lucafurrer** [comment]
fixed by #303 and #302
